### PR TITLE
add docs for compute viscosity functions

### PR DESCRIPTION
**Pull request by tduretz**
Add docs to `compute_viscosity_**` functions.

### CODE REVIEW

**Comment by albert-de-montserrat** [comment]
Thanks for the contribution @tduretz  ! 

It'd be great if you could add a MarkDown file in the `docs/src/man` folder as well.  And add it to the list in `docs/make.jl`.

File: src/Viscosity/Viscosity.jl [hunk 1]
```diff
@@ -10,6 +10,11 @@ export compute_viscosity_εII,
 @inline _viscosity(τII, εII) = τII / (2  * εII)
 
 # compute effective "creep" viscosity from strain rate tensor
+"""
+    compute_viscosity_εII(s::AbstractConstitutiveLaw, εII, kwargs...)
+
+Compute effective viscosity given a 2nd invariant of the deviatoric strain rate tensor and an extra paramaters undet the from of a named tuple, e.g., (;T=T)
```
Comment (albert-de-montserrat) [inline src/Viscosity/Viscosity.jl:16]: Typos

File: src/Viscosity/Viscosity.jl [hunk 2]
```diff
@@ -18,6 +23,11 @@ export compute_viscosity_εII,
 end
 
 # compute effective "creep" viscosity from deviatoric stress tensor
+"""
+    compute_viscosity_τII(s::AbstractConstitutiveLaw, τII, kwargs...)
+
```
Comment (albert-de-montserrat) [inline src/Viscosity/Viscosity.jl:28]: Typos

**Review by albert-de-montserrat — CHANGES_REQUESTED** [review]

File: src/Viscosity/Viscosity.jl (hunk 1 above)
Comment (tduretz) [reply src/Viscosity/Viscosity.jl]: fixed

File: src/Viscosity/Viscosity.jl (hunk 2 above)
Comment (tduretz) [reply src/Viscosity/Viscosity.jl:28]: fixed

**Review by albert-de-montserrat — APPROVED** [review]
Great thanks! Looks ready to be merged.

**Pull request merged**